src/parsers/pm_io/psse.jl

```diff
@@ -326,8 +326,11 @@ function _psse2pm_generator!(pm_data::Dict, pti_data::Dict, import_all::Bool)
             sub_data["r_source"] = pop!(gen, "ZR")
             sub_data["x_source"] = pop!(gen, "ZX")
 
-            if sub_data["gen_status"] == 1 && sub_data["pg"] == 0.0 &&
-               pm_data["bus"][sub_data["gen_bus"]]["bus_type"] == 2
+            if sub_data["gen_status"] == 1 &&
```

Comment:
@mcllerena we should wrap this up in a function called `_is_synch_condenser` this is too long to understand what the conditional is.